src/content/basics/automatic-termination-of-bad-nodes/index.md

```diff
@@ -46,3 +50,23 @@ spec:
 ```
 
 If you want to disable the feature you must remove the annotation from the [`AWSCluster`](/reference/cp-k8s-api/awsclusters.infrastructure.giantswarm.io/) custom resource.
+
+### Azure
+
+To enable it, you have to edit the [`Cluster`](/reference/cp-k8s-api/clusters.cluster.x-k8s.io/) resource of your cluster using the [Control Plane Kubernetes API](/basics/api/#cp-k8s-api).
+
+Make sure the resource has the `alpha.node.giantswarm.io/terminate-unhealthy` annotation. The value can be anything you like, as only the presence of that annotation is checked. Here is an example:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  annotations:
+    alpha.node.giantswarm.io/terminate-unhealthy: "true"
```

Comment:
Is it required that the value here is `"true"`? If not, I would suggest to make it consistent with line 41. In which way you align is up to you.

Reply:
done